tests/foreman/destructive/test_installer.py

```diff
@@ -147,3 +149,44 @@ def test_positive_installer_certs_regenerate(target_sat):
     )
     assert result.status == 0
     assert 'FAIL' not in target_sat.cli.Base.ping()
+
+
+def test_positive_installer_puma_worker_count(target_sat):
+    """Installer should set the puma worker count and thread max without having to manually
+    restart the foreman service.
+
+    :id: d0e7d958-dd3e-4962-bf5a-8d7ec36f3485
+
+    :steps:
+        1. Check how many puma workers there are by default (9)
+        2. Change answer's file to have 5 puma workers
+        3. Run satellite-installer --foreman-foreman-service-puma-workers 5 --foreman-foreman-service-puma-threads-max 5
+
+    :expectedresults: aux should show there are only 5 puma workers after installer runs
+
+    :BZ: 2025760
+
+    :customerscenario: true
+    """
+    result = target_sat.execute('ps aux | grep -v grep | grep -e USER -e puma')
+    count = 0
+    for line in result.stdout.splitlines():
+        if 'puma: cluster worker' in line:
+            count += 1
```

Comment:
likely easier to do it this way
```suggestion
    count = int(target_sat.execute('pgrep --full "puma: cluster worker" | wc -l').stdout)
```